lib/heroics.rb

```diff
@@ -12,7 +12,7 @@ module Heroics
 
   def default_configuration(&block)
     block ||= lambda { |c| }
-    Heroics::Configuration.defaults.tap(&block)
+    Heroics::Configuration.defaults(&block).tap(&block)
```

Comment:
Does it instead make sense to call something like `Heroics::Configuration.set_defaults(&block)` and that new class method would be the thing that calls `defaults` and memoizes it?